homeassistant/components/simplisafe/__init__.py

```diff
@@ -110,6 +110,13 @@
 )
 
 
+@callback
+def _async_get_refresh_token(hass, config_entry):
+    """Retrieve a refresh token from the config entry."""
+    entry = hass.config_entries.async_get_entry(config_entry.entry_id)
```

Comment:
We're passing in the config entry. Why do we need to look it up again?

Reply:
Wow. I must have not slept well that night. 😂😂😂😂